core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -541,9 +541,17 @@ class KafkaServer(
             }.toMap
         }
 
-        val fetchManager = new FetchManager(Time.SYSTEM,
-          new FetchSessionCache(config.maxIncrementalFetchSessionCacheSlots,
-            KafkaServer.MIN_INCREMENTAL_FETCH_SESSION_EVICTION_MS))
+        // The FetchSessionCache is divided into config.numIoThreads shards, each responsible
+        // for sessionIds falling in [Max(1, shardNum * sessionIdRange), (shardNum + 1) * sessionIdRange)
+        val sessionIdRange = Int.MaxValue / config.numIoThreads
+        val fetchSessionCaches = Range(0, config.numIoThreads)
```

Comment:
same comment about `Range` vs `to`

Reply:
I've used `until` because I needed an exclusive Range